Update NVDA Icon

feerrenrut opened the pull request:
### Summary of the issue:
New icon should have better contrast so that is shows up better on Windows 10 taskbar / start menu. A more modern "flat" design.

### Description of how this pull request fixes the issue:
Includes the new icon and svg source for the icon. Description of the change to the icon:

The previous dark grey shaded icon has been in use for nearly 10 years. It was first created for Windows XP, which had a light grey system tray which showed up the dark grey logo very well.  Recent versions of Windows have moved towards a darker system tray.  There is very little contrast with the existing NVDA icon.  The updated icon features a similar logo in flat white, on a solid purple background. This ensures it will be visible on any colour background, and uses the purple from the NV Access logo.

The logo is a stylised blend of the letters NVDA. It has a vertical line on the left with a little hook at the bottom. Then from the top, a straight diagonal line comes down to the right. Next, a line curves in a semi-circle out to the right then back up to the top, and finally there is a little hook down to finish. The vertical and diagonal lines make the first part of the letter N and also the letter A (without the cross bar). The diagonal and curved lines form a V, and the curve is the right part of the D.

### Testing performed:
Locally built the launcher and installed. Inspected the icon in the taskbar / tray / start menu.

### Change log entry:
* Changes
`The NVDA logo has been updated. The updated NVDA logo is a stylised blend of the letters NVDA in white, on a solid purple background. This ensures it will be visible on any colour background, and uses the purple from the NV Access logo. (#7446)`

feerrenrut commented:
@Qchristensen Could you test this out on your systems as well: https://ci.appveyor.com/project/NVAccess/nvda/build/try-updateNVDAIcon-14237,beeb0305/artifacts

jcsteh commented:
Once #7462 is merged to master, this PR should be rebased and the icon should be added to the NVDA repo, rather than updating miscDeps.

feerrenrut committed:
Update NVDA icon

See PR #7446

Include the source `svg` so that future modifications can be made more
easily.

The previous dark grey shaded icon has been in use for nearly 10 years.
It was first created for Windows XP, which had a light grey system tray
which showed up the dark grey logo very well.  Recent versions of
Windows have moved towards a darker system tray.  There is very little
contrast with the existing NVDA icon.  The updated icon features a
similar logo in flat white, on a solid purple background. This ensures it
will be visible on any colour background, and uses the purple from the
NV Access logo.

The NVDA logo is a stylised blend of the letters NVDA. It has a vertical
line on the left with a little hook at the bottom. Then from the top, a
straight diagonal line comes down to the right. Next, a line curves in a
semi-circle out to the right then back up to the top, and finally there
is a little hook down to finish. The vertical and diagonal lines make
the first part of the letter N and also the letter A (without the cross
bar). The diagonal and curved lines form a V, and the curve is the right
part of the D.

feerrenrut commented:
This PR will be merged directly to master, since there is little risk caused by the change. We are also more likely to get feedback on the change once it is in master.
